libkpod/mount.go

```diff
@@ -0,0 +1,42 @@
+package libkpod
+
+import (
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+)
+
+// Mount mounts a containers layer and returns the mountpoint
+func Mount(store storage.Store, cname string, options string) (string, error) {
+	rcstore, err := store.ContainerStore()
```

Comment:
This bypasses locking and synchronization.  Use the `store` object's `Mount` method instead.